website/static/js/citationsNodeConfig.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Module that controls the citations addons node settings. Includes Knockout view-model
+ * for syncing data, and HGrid-folderpicker for selecting a folder.
+ */
+'use strict';
+
+var ko = require('knockout');
+require('knockout-punches');
+var $ = require('jquery');
+var bootbox = require('bootbox');
+var Raven = require('raven-js');
+
+var FolderPicker = require('folderpicker');
+var $osf = require('osfHelpers');
+var oop = require('js/oop');
+var FolderPickerViewModel = require('js/folderPickerNodeConfig');
+
+ko.punches.enableAll();
+/**
+ * Knockout view model for citations node settings widget.
+ */
+var CitationsFolderPickerViewModel = oop.extend(FolderPickerViewModel, {
+    constructor: function(addonName, url, selector, folderPicker) {
+        var self = this;
+        self.super.constructor.call(self, addonName, url, selector, folderPicker);
+        self.userAccountId = ko.observable('');
+    },
+    updateAccounts: function(callback) {
```

Comment:
`callback` shouldn't be necessary if you return the `jqXHR`, which is a promise.